src/hooks/transactions/useTransactionsTracker.ts

```diff
@@ -21,11 +25,22 @@ export function useTransactionsTracker(props?: TransactionsTrackerType) {
 
   useRegisterWebsocketListener(onMessage);
 
-  useEffect(() => {
+  const setPollingInterval = async () => {
+    const websocketUrl = await getWebsocketUrl(network.apiAddress);
+
+    if (websocketUrl != null) {
```

Comment:
I think the condition here should be:
- do not set interval if subscription to WS event was successful

(ws addres might be down so backup mechanism should enter)